plugins/source/airtable/src/tables.ts

```diff
@@ -78,17 +68,7 @@ const airtableFieldToArrowField = (field: APIField): DataType => {
     }
     case APIFieldType.createdTime:
     case APIFieldType.lastModifiedTime: {
-      switch (field.options.result.type) {
-        case 'date': {
-          return new Date_(DateUnit.DAY);
```

Comment:
We don't want to use Arrow Date as most (all?) of our destinations map it to string